docs/excel/custom-functions-overview.md

```diff
@@ -360,6 +360,50 @@ function secondHighest(values){
 }
 ```
 
+## Discovering cells that invoke custom functions
+
+You may wish to discover the cell that invoked a custom function. Common scenarios for using `caller.address` include:
+
+- Formatting ranges: You can use `caller.address` as the key of the cell to store information in [`AsyncStorage`](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-runtime#storing-and-accessing-data). Then, use [the `onCalculated` event](https://docs.microsoft.com/en-us/javascript/api/excel/excel.worksheet?view=office-js#oncalculated) in Excel to load the key from `AsyncStorage`.
+- Displaying cached values: If your function is being used offline, you can display stored cached values from `AsyncStorage`. Display using the `onCalculated` event.
+- Reconciliation: You can use `caller.address` to discover an origin cell to help you reconcile where processing is occurring.
+
+The information about a cell's address is exposed only if `requiresAddress` is marked as `true` in the function's JSON metadata file. As an example of the JSON metadata, see the following example:
+
+```JSON
+{
+   "id": "ADDTIME",
+   "name": "ADDTIME",
+   "description": "Display the date right now and add the amount of hours to it designated by the parameter",
+   "helpUrl": "http://www.contoso.com",
+   "result": {
+      "type": "number",
+      "dimensionality": "scalar"
+   },
+   "parameters": [
+      {
+         "name": "Additional time",
+         "description": "Amount of hours to increase current date by",
+         "type": "number",
+         "dimensionality": "scalar"
+      }
+   ],
+   "options": {
+      "requiresAddress": true
+   }
+}
+```
+
+In the script file (./src/customfunctions.js or ./src/customfunctions.ts), add a function similar to the `getAddress` in the following sample to get an address for a cell which issued a call. This function may take parameters, as shown in the following sample as `parameter1`, but the last parameter will always be `invocationContext`, an object containing the cell's location that Excel passes down when `requiresAddress` is marked as `true` in your JSON metadata file.
```

Comment:
This first sentence sounds clunky. Maybe there's a way to rewrite this block to say something like "If you're using JavaScript, add the following function to **./src/customfunctions.js**.